WordPress/src/main/res/values/strings.xml

```diff
@@ -4336,4 +4336,6 @@ translators: %s: Select control option value e.g: "Auto, 25%". -->
     <string name="jetpack_feature_card_menu_item_remind_me_later">Remind me later</string>
     <string name="jetpack_feature_card_menu_item_hide_this">Hide this</string>
     <string name="gutenberg_native_featured" tools:ignore="UnusedResources" a8c-src-lib="gutenberg">Featured</string>
+
+    <string name="install_play_store_to_get_jetpack">Please install google play store to get the Jetpack app</string>
```

Comment:
Hi, @AjeshRPai! 👋 

What do you think about capitalizing the product name so it reads `Google Play Store` [as it's used by Google](https://support.google.com/googleplay/answer/190860?hl=en).